### ISSUE TITLE
nginx uwsgi server config with https

### ISSUE THREAD

**Issue by aronysidoro** [jrief/django-websocket-redis#133]
Hi @jrief, I first just want to say a big thank you for this awesome library.  I have really enjoyed using it for about 6 months now in development, and it's pretty cool how everything just "works!"

My question is, I am following this setup config:

http://django-websocket-redis.readthedocs.org/en/latest/running.html#django-with-websockets-for-redis-behind-nginx-using-uwsgi

I am having trouble because the ``WEBSOCKET_URI`` is defaulting to a protocol of `wss://` because the site is being served over **https**.  Howerver, I am getting the error:

```
failed: Error during WebSocket handshake: Unexpected response code: 500
Websocket connection is broken!
```

Do you know why this may be happening?  My site is working fine with uWSGI and Nginx over https.  I followed the "read the docs" exactly, but still stuck on this error.

Any help is much appreciated. Thank you in advance.

**Comment by jrief** [comment]
Hi @aronysidoro 
difficult to say. The uWSGI setup is quite tricky and there are sooooo many options which can be wrong.
I'd first try with uwsgi in stand-anlone mode, and without https. Run it on the command line to see errors. Starting from there, evolve into your configuration.

**Comment by aronysidoro** [comment]
@jrief,

Following your docs [here](http://django-websocket-redis.readthedocs.org/en/latest/running.html#django-with-websockets-for-redis-behind-nginx-using-uwsgi), what I did was within my `uwsgi_websocket.ini` file, I pointed to the `wsgi_websocket.py` file that you said to use like so:

```
wsgi-file = /opt/django/wsgi_websocket.py
```

My previous unworking config looked like this.  I had put everything in a `.wsgi` file, and the `uwsgi log` was telling me "no such application":

```
module = wsgi_websocket.wsgi:application
```

That was the only change in config needed.  Thank you again for providing this awesome library.

Cheers!
